omero/import_scripts/Populate_Metadata.py

```diff
@@ -84,24 +92,24 @@ def populate_metadata(client, conn, script_params):
 
 def run_script():
 
-    data_types = [rstring('Plate'), rstring('Screen')]
+    data_types = [rstring(otype) for otype in OBJECT_TYPES]
     client = scripts.client(
         'Populate_Metadata.py',
         """
-    This script processes a csv file, attached to a Screen or Plate,
-    converting it to an OMERO.table, with one row per Well.
+    This script processes a csv file, attached to a container,
+    converting it to an OMERO.table, with one row per Image or Well.
     The table data can then be displayed in the OMERO clients.
     For full details, see
     http://help.openmicroscopy.org/scripts.html#metadata
         """,
         scripts.String(
             "Data_Type", optional=False, grouping="1",
             description="Choose source of images",
-            values=data_types, default="Plate"),
+            values=data_types, default=OBJECT_TYPES[0]),
 
         scripts.List(
             "IDs", optional=False, grouping="2",
-            description="Plate or Screen ID.").ofType(rlong(0)),
+            description="Container ID.").ofType(rlong(0)),
 
         scripts.String(
             "File_Annotation", grouping="3",
```

Comment:
Not related to this PR, but since you're making changes, perhaps you could change this to a ```Long``` input instead of a string?